scene/gui/texture_progress.cpp

```diff
@@ -189,30 +191,30 @@ void TextureProgress::draw_nine_patch_stretched(const Ref<Texture> &p_texture, F
 		width_texture = MIN(width_texture, first_section_size + middle_section_size + last_section_size);
 
 		switch (mode) {
-			case FILL_LEFT_TO_RIGHT: {
-				src_rect.size.x = width_texture;
-				dst_rect.size.x = width_filled;
-				bottomright.x = last_section_size;
-			} break;
-			case FILL_RIGHT_TO_LEFT: {
-				src_rect.position.x += src_rect.size.x - width_texture;
-				src_rect.size.x = width_texture;
-				dst_rect.position.x += width_total - width_filled;
-				dst_rect.size.x = width_filled;
-				topleft.x = last_section_size;
-			} break;
-			case FILL_TOP_TO_BOTTOM: {
-				src_rect.size.y = width_texture;
-				dst_rect.size.y = width_filled;
-				bottomright.y = last_section_size;
-			} break;
-			case FILL_BOTTOM_TO_TOP: {
-				src_rect.position.y += src_rect.size.y - width_texture;
-				src_rect.size.y = width_texture;
-				dst_rect.position.y += width_total - width_filled;
-				dst_rect.size.y = width_filled;
-				topleft.y = last_section_size;
-			} break;
+		case FILL_LEFT_TO_RIGHT: {
```

Comment:
Why change the indentation? The new style you're using does not follow our guidelines: http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html